frontend/src/metabase/visualizations/components/settings/ChartSettingInputSuggestion.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useRef } from "react";
+import TippyPopoverWithTrigger from "metabase/components/PopoverWithTrigger/TippyPopoverWithTrigger";
+
+import {
+  SuggestionInput,
+  SuggestionContainer,
+  Suggestion,
+} from "./ChartSettingInputSuggestion.styled";
+
+interface ChartSettingInputSuggestionProps {
+  value: string;
+  onChange: (value: string) => void;
+  id?: string;
+  keys?: string[];
+}
+
+const ChartSettingInputSuggestion = ({
```

Comment:
I'm wondering why you did not use our `Select` component here because mainly because it has keyboard navigation built-in which is natural in such cases. If there is a stong UX arguments against it, keyboard navigation would be still nice to have and there is a hook for that `frontend/src/metabase/hooks/use-list-keyboard-navigation/use-list-keyboard-navigation.ts`